Makefile

```diff
@@ -423,3 +423,23 @@ generate-flag-testdata:
 
 install_kubectl_kind:
 	./tools/get_kubectl_kind.sh
+
+
+
+# Scripts related to test coverage
+PACKAGE ?= ./go/...
+
+coverage:
+	go install golang.org/x/tools/cmd/cover
+	go test -coverprofile=coverage.out $(PACKAGE)
```

Comment:
Just a thought, but I often find myself using the `-v` flag too as it helps find failing tests. Might be worth adding